### PR TITLE
Add myself as a co-maintainer of git-worktree-switcher

### PR DESCRIPTION
**Pull request by mateusauler**
As it says on the tin.
Let's hope this works.

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -13533,6 +13533,13 @@
     githubId = 28183516;
     name = "Mateusz Słodkowicz";
   };
+  mateusauler = {
+    email = "mateus@auler.dev";
+    github = "mateusauler";
+    githubId = "24767687";
+    name = "Mateus Auler";
+    keys = [ { fingerprint = "A09D C093 3C37 4BFC 2B5A  269F 80A5 D62F 6EB7 D9F0"; } ];
```
Comment (jiriks74) [inline maintainers/maintainer-list.nix]: Format the file. The `keys` attribute will triger `nixpkgs`' Ci/Cd check. Go into a `nix-shell` from the root of `nixpkgs` and run `nix-fmt maintainers/maintainer-list.nix`.

PS: The commands are from memory. If they don't work look at the docs or one of the Actions where I failed to format things (it described there as well).

> [!Note]
> Don't try to format everything, it won't finish today XD

**Review by jiriks74 — CHANGES_REQUESTED** [review]
You'll need to fix some formatting otherwise it looks OK. The maintainers file is sorted so you only need to format it.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (jiriks74) [reply maintainers/maintainer-list.nix]: I got the commands for you from [a failed action](https://github.com/NixOS/nixpkgs/actions/runs/11803569498/job/32881816498):

```
Please go to the Nixpkgs root directory, run `nix-shell`, then:
nixfmt 'maintainers/maintainer-list.nix'
```

Comment (mateusauler) [reply maintainers/maintainer-list.nix]: Forgot about that :P
I tried running nixfmt, but it did nothing.
So I did it manually. Does it look good?

**Review by jiriks74 — APPROVED** [review]
Seems OK now

**Comment by jiriks74** [comment]
@mateusauler 

Seems like I can't modify the commit history without either pulling the commits back from your tree or breaking your signature.

Since Nix requires a working signature for the maintainers addition commit I'd need you to pull the new tree and add yourself again. Would you be able to do so?

Sorry, I haven't found a better solution yet. IDK why git modifies the comit even if I specify it as `pick` and not `edit`...

**Comment by mateusauler** [comment]
No worries! I believe it modifies the commits because their parents are changing so the parent's hash changes as well.

I'll open another PR with the same commit and once you finish tidying the commits, you can merge it.

**Comment by jiriks74** [comment]
@mateusauler It's cleaned up now so you can yourself back to the maintainers.

**Comment by jiriks74** [comment]
Nope, there's a formatting issue

**Comment by jiriks74** [comment]
Now you can

**Comment by jiriks74** [comment]
God, modifying git history is nerve-racking

**Comment by mateusauler** [comment]
Yeah, I've been through that...

**Pull request merged**